.travis.yml

```diff
@@ -64,6 +64,7 @@ env:
         - CONF="--disable-abi-compat"
         - CONF="--enable-schedule-sp"
         - CONF="--enable-schedule-iquery"
+        - CONF="--enable-dpdk-zero-copy"
```

Comment:
Do we have DPDK in GitHub where Travis can find it? If not this option will fail. Also how is the --with-dpdk-path config option specified in these automated builds?